pinax/stripe/models.py

```diff
@@ -197,10 +197,31 @@ class TransferChargeFee(models.Model):
     created_at = models.DateTimeField(default=timezone.now)
 
 
+class UserAccount(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             related_name="user_accounts",
+                             related_query_name="user_account",
+                             on_delete=models.CASCADE)
+    account = models.ForeignKey("pinax_stripe.Account",
+                                related_name="user_accounts",
+                                related_query_name="user_account",
+                                on_delete=models.CASCADE)
+    customer = models.ForeignKey("pinax_stripe.Customer",
+                                 related_name="user_accounts",
+                                 related_query_name="user_account",
+                                 on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("user", "account")
```

Comment:
It should be noted that we went with having `account` here mainly/only because of the `unique_together`.
After all the account is available on the `customer`.

I think there should be validation in place to ensure that `self.account == self.user.stripe_account`, e.g. in `clean()`?!